leshan-core/src/main/java/org/eclipse/leshan/core/node/codec/LwM2mNodeDecoder.java

```diff
@@ -210,7 +247,83 @@ private static LwM2mNode parseTlv(Tlv[] tlvs, LwM2mPath path, LwM2mModel model)
             }
         }
     }
+    
+    private static LwM2mNode parseJSON(LwM2mJsonObject jsonObject, LwM2mPath path, LwM2mModel model) throws InvalidValueException {
+        LOG.trace("Parsing JSON content for path {}: {}", path, jsonObject);
+
+        if (path.isObject()) {
+           // TODO 
+           // If bn is present will have multiple object instances in JSON the payload
+           // In case of JSON contains ObjLnk -> this should return List<LwM2mNode> ???
+        	 throw new InvalidValueException("Not yet implemented: " + " Object Path ", path);
```

Comment:
Where to find InvalidStateException?
I don't think is part of the JDK, do you mean IllegalStateException?